pages/agent/v3/aws/secrets_manager.md

```diff
@@ -0,0 +1,50 @@
+# Storing your Buildkite Agent token in AWS Secrets Manager
+
+If you prefer to store your Buildkite Agent token as an AWS Secrets Manager secret,
```

Comment:
Maybe it's worth a short preamble about the elastic stack having built-in support for reading the  agent token from Parameter Store, as a foundation for discussing why using Secrets Manager might be an interesting alternative for some folks.